source/leuven/superdome_quick_start.rst

```diff
@@ -1,77 +1,42 @@
 Superdome quick start guide
 ===========================
 
-The :ref:`Superdome <Superdome hardware>` is a shared memory machine, i.e.,
-it can be used to run multi-threaded application that require large amounts
-of RAM.
-
-
-How to connect to Superdome?
-----------------------------
-
-Superdome does not have a dedicated login node, so in order to work with the
-system users can connect to the same login node used for Genius.
+The :ref:`Superdome <Superdome hardware>` is a shared memory machine
+which is intended for applications requiring large amounts of RAM
+on a single host.
 
 .. include:: tier2_hardware/genius_login_nodes.rst
 
 How to run a job on Superdome?
 ------------------------------
 
-Jobs can be submitted from the login node, however, there are a few crucial
-differences.  To submit jobs to Superdome you need to
+To submit jobs to Superdome from a Genius login node you need to select either
+the ``superdome`` or ``superdome_long`` Slurm partition, depending on whether
+your job needs less than 3 days or less than 7 days.
 
-- specify the partition ``superdome``,
-- specify the queue ``qsuperdome``, and
-- use ``-L`` for specifying resources.
+The CPU resources on Superdome are furthermore allocated on the basis of entire
+sockets and not on the basis of individual cores as on other partitions.
+Since the 112 cores in Superdome are grouped in 8 sockets (see the
+:ref:`Superdome hardware page<Superdome hardware>`), this means that all
```

Comment:
This link is not correctly rendering.

Reply:
Could you double check? When I render it and view the page the link looks fine and points to expected destination.

Reply:
yes, it renders correctly when I fully make the docs locally on my laptop.

Reply:
For me this is also ok.